app/javascript/controllers/playlist_controller.js

```diff
@@ -0,0 +1,59 @@
+import { Controller } from 'stimulus'
+
+export default class extends Controller {
+  static targets = ['credits', 'sidebar', 'content']
+
+  initialize() {
+    this.resize()
+  }
+
+  resize() {
+    if (this.isCoverView()) {
+      if (this.hasCredits()) {
+        this.alignCredits()
+      }
+      this.roundBottomRightCorner()
+    }
+  }
+
+  alignCredits() {
+    if (this.isTwoColumn()) {
+      this.creditsTarget.style.width = this.contentTarget.clientWidth + "px"
+      this.element.style.paddingBottom = this.offset() + "px"
+    } else {
+      this.creditsTarget.style.width = "100%"
+      this.element.style.paddingBottom = "0px"
+    }
+  }
+
+  roundBottomRightCorner() {
+    if ( this.heightDifference() > 0 ) {
+      this.sidebarTarget.classList.add('rounded_bottom_right_corner')
+     } else {
+        this.sidebarTarget.classList.remove('rounded_bottom_right_corner')
+    }
+  }
+  
+  hasCredits() {
+    return this.hasCreditsTarget
+  }
+
+  isCoverView() {
+    return document.body.classList.contains('cover_view')
```

Comment:
When possible, it's best to only query the DOM via stimulus targets inside the controller scope. I would:

1) make `.large_cover_and_credits` a target 
2) use the [hasABCTarget](https://stimulusjs.org/reference/targets#properties)
3) Use `this.hasCoverTarget` instead of a new method

Reply:
👍🏻